nemo/core/optim/lr_scheduler.py

```diff
@@ -824,6 +824,9 @@ def prepare_lr_scheduler(
     if add_max_args_flag and scheduler_config.get('name', '') != "ExponentialLR":
         scheduler_args['max_steps'] = max_steps
 
+    if scheduler_config.get('name', '') == "CyclicLR":
+        del scheduler_args['max_steps']
```

Comment:
Does this need to go here? Can it go in `configure_optimizers` or do we need this every time we use "CyclicLR" ?